File: src/net/eduard/api/command/api/ApiUnloadWorldCommand.java
package net.eduard.api.command.api;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.Mine;
import net.eduard.api.lib.manager.CommandManager;

public class ApiUnloadWorldCommand extends CommandManager {
	public ApiUnloadWorldCommand() {
		super("unloadworld", "descarregarmundo", "desligarmundo");
		setUsage("/api unload <world>");
		setDescription("Descarrega um mundo carregado no servidor");

	}

	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
		if (args.length == 1) {
			sendUsage(sender);
		} else {
			if (Mine.existsWorld(sender, args[1])) {
				Mine.unloadWorld(args[1]);
				sender.sendMessage("§bEduardAPI §aVoce descarregou o mundo §2" + args[1]);
			}
		}

		return true;
	}

}

File: src/main/java/net/eduard/api/command/api/ApiUnloadWorldCommand.kt
package net.eduard.api.command.api

import net.eduard.api.lib.manager.CommandManager
import net.eduard.api.lib.modules.Mine
import org.bukkit.command.Command
import org.bukkit.command.CommandSender

class ApiUnloadWorldCommand : CommandManager("unloadworld", "descarregarmundo", "desligarmundo") {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        if (args.isEmpty()) {
            sendUsage(sender)
            return true
        }
        if (Mine.existsWorld(sender, args[0])) {
            Mine.unloadWorld(args[0], true)
            sender.sendMessage("§bEduardAPI §aVoce descarregou o mundo §2" + args[0])
        }
        return true
    }

    init {
        usage = "/api unloadworld <world>"
        description = "Descarrega um mundo carregado no servidor"
    }
}